src/AdvApi32.Desktop/AdvApi32+TokenType.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+
+    /// <content>
+    /// Exported functions from the AdvApi32.dll Windows library
+    /// that are available to Desktop apps only.
+    /// </content>
+    public static partial class AdvApi32
+    {
+        /// <summary>
+        /// The <see cref="TokenType"/> enumeration contains values that differentiate between a primary token and an
+        /// impersonation token.
+        /// </summary>
+        public enum TokenType
```

Comment:
This enumeration is named `TOKEN_TYPE` on MSDN : https://msdn.microsoft.com/en-us/library/windows/desktop/aa379633(v=vs.85).aspx